locales/en/messages.json

```diff
@@ -1019,7 +1019,7 @@
         "message": "SPI RX support"
     },
     "featureESC_SENSOR": {
-        "message": "Use KISS/BLHeli_32 ESC telemetry as sensor"
+        "message": "Use KISS/BLHeli_32 ESC <b>on a separate wire</b> telemetry as sensor"
```

Comment:
Should this be 'Use KISS/BLHeli_32 ESC telemetry over a separate wire as sensor'? Or I'd probably even drop the 'as sensor', as it is fairly redundant.

Reply:
Right, changed.